rabbitmq-http-connector/main.go

```diff
@@ -44,7 +44,11 @@ func (conn rabbitMQConnector) consumeMessage() {
 	}
 
 	forever := make(chan bool)
-	sem := make(chan int, 10) // Process maximum 10 messages concurrently
+	concurrent = os.Getenv("CONCURRENT")
```

Comment:
```suggestion
	concurrent := os.Getenv("CONCURRENT")
```

Reply:
Oops, sorry that I still not get used to golang syntax. 😅